app/assets/javascripts/views/story_view.js

```diff
@@ -254,13 +256,33 @@ module.exports = FormView.extend({
     this.$el.appendTo(this.model.get('column'));
   },
 
+  setStoryDetailRequest: function() {
+    const shouldRequest = !this.model.attributes.isFetched;
+    if (shouldRequest) {
+      this.fetchStoryDetails();
+    }
+  },
+
   startEdit: function(e) {
     if (this.eventShouldExpandStory(e)) {
       this.model.set({editing: true, editingDescription: false, clickFromSearchResult: this.$el.hasClass('searchResult')});
+      this.setStoryDetailRequest();
       this.removeHoverbox();
+      this.render();
     }
   },
 
+  fetchStoryDetails: function() {
+    var that = this;
```

Comment:
You are using [arrow functions](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Functions/Arrow_functions) so I believe you don't need this `that` var.

Reply:
I thought that too. But when I remove the that = this line, for some reason the code stop working.

Reply:
I've tested on my environment and this following code worked for me:
```js
fetchStoryDetails: function () {
  return axios
    .get(`/projects/${this.model.collection.project.id}/stories/${this.model.id}`)
    .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
    .then(({ story }) => {
      this.model.set(story)
      this.model.set({ isFetched: true })
    })
},
```
🤔